theano/tensor/nnet/abstract_conv.py

```diff
@@ -1,5 +1,5 @@
 """
-Abstract conv interface
+Abstract conv interface updated
```

Comment:
Undo this. You didn't change the interface, so it isn't updated. You updated the file, this isn't the same.